### ISSUE TITLE
It is not possible to create an Enclosure Group although it is documented

### ISSUE THREAD

**Issue by russellseymour** [HewlettPackard/terraform-provider-oneview#46]
In the README it states that the `oneview_enclosure_group` resource can be used to create an Enclsoure Group. However when this is added to a template and TerraForm executed with the OneView plugin the following error is seen:

`Error: oneview_enclosure_group.enclosure_group_1: Provider doesn't support resource: oneview_enclosure_group`

This is using the following template structure:

```
resource "oneview_enclosure_group" "enclosure_group_1" {
  name = "my_enclosure_group"
}
```

This maybe related to issue https://github.com/HewlettPackard/terraform-provider-oneview/issues/38.

The plugin was built from `9aa276bafe873463b85ae1abce7c13ed8ad506ca` version of the repo.

**Comment by 5mzJyf6m** [comment]
Is this scheduled to be fixed?

**Comment by jyothis-gireesan** [comment]
@5mzJyf6m It is currently available in https://github.com/HewlettPackard/terraform-provider-oneview/tree/enclosuregroup branch. It is waiting for approval form Hashicorp and once it is done, it will be available in the master branch.

These resources and its operations will be available in our upcoming release for the terraform along with use cases and examples which supports till OneView 4.1 or Oneview API 800.

**Comment by 5mzJyf6m** [comment]
@jyothis-gireesan Thanks for the response. Is the new release scheduled?

**Comment by jyothis-gireesan** [comment]
@5mzJyf6m The release date is not yet set since we are facing external delays due to the approvals for the PRs are getting delayed from Hashicorp.

Will be doing the release as soon as the approvals come through.

**Comment by AsisBagga** [comment]
Since we have received necessary approval form Hashicorp it is  available in the master branch.
I'll proceed with closing this issue. For any other issues, please proceed with closing a new issue.
Thanks.
